development/compiling/compiling_for_x11.rst

```diff
@@ -123,6 +123,11 @@ following parameters:
 Note that cross compiling for the opposite bits (64/32) as your host
 platform is not always straight-forward and might need a chroot environment.
 
+.. note:: If you are compiling Godot for production use and not to hack
```

Comment:
I suggest some changes for conciseness and readability (and fixing the typo in `executable`):

```
.. note:: If you are compiling Godot for production use, then you can
          make the final executable smaller and faster by adding the
          SCons option ``target=release_debug``.
```

Also, since the note pertains to the editor, I would place it at the bottom of the **Compiling** section rather than in **Building export templates**.